src/gui_common/DraggableScrollContainer.cs

```diff
@@ -79,8 +79,7 @@ public override void _Ready()
     {
         base._Ready();
 
-        if (ContentPath == null)
-            throw new InvalidOperationException("Path to child control is expected");
+        ContentPath ??= GetChild(0).GetPath();
```

Comment:
Could you write documentation on the ContentPath variable that if it is null then the path of the first child is used?